CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# How to contribute
+
+Thank you for taking time to contribute to this repository! This guide should help you effectively do so by following the systems already in place for maintaining the repository.
+
+## Table of contents
+
+* [Code of conduct](#code-of-conduct)
+* [Raising an issue](#raising-an-issue)
+* [Making a pull request](#making-a-pull-request)
+* [Coding standards](#coding-standards)
+  * [Code style](#code-style)
+  * [Testing](#testing)
+* [Updating .gitignore file](#updating_.gitignore_file)
+* [External resources](#external-resources)
+
+## Code of conduct
+
+This repository and everyone contributing to is subject to follow the adopted [code of conduct](CODE_OF_CONDUCT.md). Should you choose to contribute, you will be expected to uphold this code. Please report any violations to [@gkapfham](https://github.com/gkapfham) or [@philmcminn](https://github.com/philmcminn).
+
+## Raising an issue
+
+First, check the [Issue Tracker](https://github.com/AVMf/avmf/issues) to make sure it has not already been made there. If it would be a new issue, go ahead and [raise it](https://github.com/AVMf/avmf/issues/new)! We just ask that you:
+* give it a descriptive title
+* provide any relevant information
+* add any labels that apply to it
+* reference any other related issues
+
+## Making a pull request
+
+When making a [pull request](https://github.com/AVMf/avmf/pulls), we ask that you do the following:
+* give it a descriptive title
+* include language that will close all issues it is proposing fixes for
+* describes any issues that it is proposing to fix that are not in the issue tracker to give a greater idea as to why the pull request should be approved
```

Comment:
A quick change! Can you use `describe` instead of `describes`? Same with the couple ones following.